scrapd/core/apd.py

```diff
@@ -536,6 +539,29 @@ def parse_crashes_field(page):
     return match_pattern(page, crashes_pattern)
 
 
+def parse_date_field(page):
+    """
+    Extract the date from the content of the fatality page.
+
+    :param str page: the content of the fatality page
+    :return: a string representing the date.
+    :rtype: str
+    """
+    date_pattern = re.compile(
+        r'''
+        >Date:          # The name of the desired field.
+        .*\s{2,}        # Any character and whitespace.
+        (?:</strong>)?  # Non-capture (literal match).
+        ([^<]*)         # Capture any character except '<'.
+        <               # Non-capture (literal match)
+        ''',
+        re.VERBOSE,
+    )
+    date = match_pattern(page, date_pattern)
+    date = search_dates(date)
```

Comment:
If `search_dates()` does not find any date it will return `None`, so you should account for that too.

Reply:
I was accounting for that on the return line (L562) 
`return date[0][1].strftime("%B %d, %Y") if date else ''`
Would you like the return to be `None` instead of `''` if date is `None`?

Regarding the format, I didn't realize there was an existing format, as the regex seemed to be matching/returning whatever APD entered in the date field, regardless of format. 
For future reference, where would I have looked to find the existing format?

Reply:
Ah, yes, you're right, I missed the catch on line 562. So you can disregard my comment.

Regarding the format, the regex was returning the raw string, and it was cleaned up later in the code. The clean up function is here: https://github.com/scrapd/scrapd/blob/master/scrapd/core/date_utils.py#L35-L47